ChangeLog

```diff
@@ -1,15 +1,15 @@
-7.0.6 -- 2024-06-26
+7.0.6 -- 2024-06-28
 
+Security #7105: http2: oom from duplicate headers (7.0.x backport)(CVE 2024-38535)
```

Comment:
Jason and I were discussing yesterday to add the severity as well
`Security #7105: http2: oom from duplicate headers (7.0.x backport)(CRITICAL - CVE 2024-38535)` or something, can you update the tooling to generate that and update it here?

Reply:
yes